### PR TITLE
@kanaabe => DraftJS key commands

### PR DESCRIPTION
**Pull request by eessex**
Closes https://github.com/artsy/positron/issues/971

### CODE REVIEW

File: client/apps/edit/components/section_text/index.coffee [hunk 1]
```diff
@@ -214,13 +216,38 @@ module.exports = React.createClass
...
+      if e.keyCode is 51   # command + 3
+        return 'header-three'
+      if e.keyCode is 191  # command + /
+        return 'custom-clear'
+      if e.keyCode is 55   # command + 7
+        return 'ordered-list-item'
+      if e.keyCode is 56   # command + 8
+        return 'unordered-list-item'
+      if e.keyCode is 75   # command + K
+        return 'link-prompt'
```
Comment (kanaabe) [inline client/apps/edit/components/section_text/index.coffee]: Curious, are these key bindings standard? I've never seen cmd + `/`, `7`, `8` before. Kind of interesting.

**Comment by kanaabe** [comment]
Looks good to me! Thanks for the tests.

File: client/apps/edit/components/section_text/index.coffee (hunk 1 above)
Comment (eessex) [inline client/apps/edit/components/section_text/index.coffee]: I pulled these commands from google docs

Comment (kanaabe) [inline client/apps/edit/components/section_text/index.coffee]: Cool, good source of truth

**Pull request merged**